lang/en.json

```diff
@@ -1,26 +1,26 @@
 {
-  "ACTOR.TypeCharacter": "Character",
-  "ACTOR.TypeMinion": "Minion",
-  "ACTOR.TypeVehicle": "Vehicle",
-  "ACTOR.TypeHomestead": "Homestead",
+  "TYPES.Actor.character": "Character",
+  "TYPES.Actor.minion": "Minion",
+  "TYPES.Actor.vehicle": "Vehicle",
+  "TYPES.Actor.homestead": "Homestead",
 
-  "ITEM.TypeAbility": "Ability",
-  "ITEM.TypeArmour": "Armour",
-  "ITEM.TypeCareer": "Career",
-  "ITEM.TypeCriticaldamage": "Critical Damage",
-  "ITEM.TypeCriticalinjury": "Critical Injury",
-  "ITEM.TypeForcepower": "Force Power",
-  "ITEM.TypeGear": "Gear",
-  "ITEM.TypeHomesteadupgrade": "Homestead Upgrade",
-  "ITEM.TypeItemattachment": "Item Attachment",
-  "ITEM.TypeItemmodifier": "Item Modifier",
-  "ITEM.TypeTalent": "Talent",
-  "ITEM.TypeShipattachment": "Ship Attachment",
-  "ITEM.TypeShipweapon": "Ship Weapon",
-  "ITEM.TypeSignatureability": "Signature Ability",
-  "ITEM.TypeSpecialization": "Specialization",
-  "ITEM.TypeSpecies": "Species",
-  "ITEM.TypeWeapon": "Weapon",
+  "TYPES.Item.ability": "Ability",
+  "TYPES.Item.armour": "Armour",
+  "TYPES.Item.career": "Career",
+  "TYPES.Item.criticaldamage": "Critical Damage",
+  "TYPES.Item.criticalinjury": "Critical Injury",
+  "TYPES.Item.forcepower": "Force Power",
+  "TYPES.Item.gear": "Gear",
+  "TYPES.Item.homesteadupgrade": "Homestead Upgrade",
+  "TYPES.Item.itemattachment": "Item Attachment",
+  "TYPES.Item.itemmodifier": "Item Modifier",
+  "TYPES.Item.talent": "Talent",
+  "TYPES.Item.shipweapon": "Ship Attachment",
+  "TYPES.Item.homesteadupgrade": "Ship Weapon",
+  "TYPES.Item.signatureability": "Signature Ability",
+  "TYPES.Item.specialization": "Specialization",
+  "TYPES.Item.species": "Species",
+  "TYPES.Item.weapon": "Weapon",

```

Comment:
Do these changes need to be replicated in the other lang files?

Reply:
........................................................yes

Reply:
I can attempt to commit these but I won't be able to reliably test since I don't speak any of these and haven't used the system enough lately to actually navigate it in a foreign language

Reply:
Probably safe enough to just update the keys to match, without changing any values.

Reply:
done

Reply:
I even fixed this

![image](https://github.com/StarWarsFoundryVTT/StarWarsFFG/assets/4709746/f1f0857b-4e73-47e5-8831-6e7bba0f641f)